packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.ts

```diff
@@ -4,60 +4,80 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import {ApiClients, DataType, Argument} from '../types'
+import {DataType, Argument} from '../types'
 import {useMutation} from '../useMutation'
-import {MutationFunction} from '@tanstack/react-query'
-
-type Client = ApiClients['shopperOrders']
+import {MutationFunction, useQueryClient} from '@tanstack/react-query'
+import {updateCache, QueryKeysMatrixElement, Client} from '../utils'
 
 export const ShopperOrdersMutations = {
     /**
      * Submits an order based on a prepared basket. The only considered value from the request body is basketId.
      * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=createOrder} for more information about the API endpoint.
      * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#createorder} for more information on the parameters and returned data type.
      */
-    CreateOrder: 'createOrder',
-    /**
-   * Adds a payment instrument to an order. 
+    CreateOrder: 'createOrder'
 
-Details:
-
-The payment instrument is added with the provided details. The payment method must be applicable for the order see GET
-/baskets/\{basketId\}/payment-methods, if the payment method is 'CREDIT_CARD' a paymentCard must be specified in the request.
-   * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=createPaymentInstrumentForOrder} for more information about the API endpoint.
-   * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#createpaymentinstrumentfororder} for more information on the parameters and returned data type.
-   */
-    CreatePaymentInstrumentForOrder: 'createPaymentInstrumentForOrder',
-    /**
-     * Removes a payment instrument of an order.
-     * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=removePaymentInstrumentFromOrder} for more information about the API endpoint.
-     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperorders.shopperorders-1.html#removepaymentinstrumentfromorder} for more information on the parameters and returned data type.
-     */
-    RemovePaymentInstrumentFromOrder: 'removePaymentInstrumentFromOrder',
-    /**
-   * Updates a payment instrument of an order.
+    // The following mutation hooks are NOT in use in the PWA Kit currently
```

Comment:
Do we want to leave lots of commented out code in our source? Perhaps this could be deleted with a link like this to leverage `develop` git history as a way to point to the code when we decide to resurrect it?

```
// Payment instrument API call is not implemented yet in PWA kit. 
// Reference: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/59d39976567c82baa9f9d393f1ac274c397b4c44/packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.ts#L23-L49
```